Makefile

```diff
@@ -368,8 +368,11 @@ clean-mailserver-systemd: ##@Easy Clean your systemd service for running a mails
 clean-mailserver-docker: ##@Easy Clean your Docker container running a mailserver
 	@cd _assets/compose/mailserver && $(MAKE) clean
 
-## TODO(samyoul) option to set and/or overwrite migration path
-migration: DEFAULT_MIGRATION_PATH := appdatabase/migrations/sql/
-migration: UNIX_TIMESTAMP := $(shell date +%s)
+UNIX_TIMESTAMP := $(shell date +%s)
```

Comment:
If you put it here it will execute every time you run any `make` target. And only two targets need it.
```
$(UNIX_TIMESTAMP)
$(shell date +%s)
```
It's the same length, just use the `shell date +%s` command directly in the target command.

Reply:
Resolved